src/clean.js

```diff
@@ -119,6 +119,11 @@ function clean(node, newObj) {
     });
   }
 
+  // for inline code, we always use the long form <?php
+  if (node.kind === "echo") {
+    newObj.shortForm = false;
+  }
```

Comment:
It is not right, prettier should not change ast, many people use difference `echo` form in difference situation. We need output echo as is.

Reply:
fine by me 👍